jiant/models.py

```diff
@@ -1160,6 +1176,69 @@ def _lm_forward(self, batch, task, predict):
             pass
         return out
 
+    def mlm_dynamic_masking(self, inputs, labels, mask_idx, tokenizer_name):
+        mlm_probability = 0.15
+        tokenizer = get_tokenizer(tokenizer_name)
+        # Masking code from https://github.com/huggingface/transformers/blob/master/examples/run_language_modeling.py
+        probability_matrix = torch.full(labels.shape, mlm_probability, device=inputs.device)
+        padding_mask = labels.eq(0)
+        probability_matrix.masked_fill_(padding_mask, value=0.0)
+
+        masked_indices = torch.bernoulli(probability_matrix).to(
+            device=inputs.device, dtype=torch.uint8
+        )
+        tokenizer_name = self.sent_encoder._text_field_embedder.tokenizer_required
+        labels, _ = self.sent_encoder._text_field_embedder.correct_sent_indexing(
```

Comment:
It looks like the labels will be modified (as a result of `correct_sent_indexing()`). It looks like the inputs aren't getting the same adjustment here. Is that intentional/correct?

Reply:
Yes because we do it here for inputs: https://github.com/nyu-mll/jiant/blob/master/jiant/huggingface_transformers_interface/modules.py#L401